rh_ui/templates/base.html

```diff
@@ -4,8 +4,28 @@
 {%- block preHeader -%}
 {%- autoescape false -%}
 
-{% set cookiesPageURL = url_for('i18n.info_pages_bp.cookies') %}
+{% set g_tag_id = config.GOOGLE_TAG_ID %}
+
+
+{% block head %}
+    {% if g_tag_id %}
+        {% set script_src_url = "https://www.googletagmanager.com/gtag/js?id=" + g_tag_id %}
```

Comment:
Should this be disabled if the user doesn't want additional cookies? Can't see anything blocking it at the moment